packages/core/src/app/address/StaticAddress.tsx

```diff
@@ -33,12 +33,23 @@ export interface StaticAddressEditableProps extends StaticAddressProps {
 interface WithCheckoutStaticAddressProps {
     countries?: Country[];
     fields?: FormField[];
+    loadAddressFields: () => Promise<CheckoutSelectors>
 }
 
 const StaticAddress: FunctionComponent<
     StaticAddressEditableProps & WithCheckoutStaticAddressProps
-> = ({ countries, fields, address: addressWithoutLocalization }) => {
+> = ({ countries, fields, loadAddressFields, address: addressWithoutLocalization }) => {
     const { isPayPalAxoEnabled, paypalConnectAddresses } = usePayPalConnectAddress();
+
+    useEffect(() => {
+        const fetchData = async () => {
+            await loadAddressFields();
+        }
+        if (!countries) {
+            fetchData();
```

Comment:
Need to fetch data here in case where countries are not fetched, i.e. on refreshing the checkout page when shipping and billing forms are filled already.

Reply:
Ahh I see, maybe then we need to move the call back to shipping step mount. As we should try and avoid making API calls from low level components as it can lead to side effects and make debugging harder.

Reply:
Moving back to shipping step mount will result in all e2e tests failing. And we really don't need that API call at Shipping step mount. We only need it there in case where billing is same as shipping and billing step is not complete, just to avoid postal code validation. 
Moreover this will be called only in case someone refreshes checkout after reaching payment step. So we need to decide whether we want to update all the recording files for e2e tests and move this to mount or go with the current solution. 
CC: @bc-peng

Reply:
We noticed this extra call while investigating why the majority of e2e tests failed in previous solutions.
The extra call is avoidable if we apply a stricter rule of making the request.
I think we should be able to run `loadAddressFields();` at any point within the shipping step, just need to figure out the conditions.